src/parse/string/shift/operand/operand.ts

```diff
@@ -24,9 +32,16 @@ export type parseOperand<
 > = s["unscanned"] extends Scanner.shift<infer lookahead, infer unscanned>
     ? lookahead extends "("
         ? state.reduceGroupOpen<s, unscanned>
-        : lookahead extends EnclosingChar
+        : lookahead extends EndEnclosingChar
```

Comment:
Definitely suss to check if `lookahead` extends something called `EndEnclosingChar` to start parsing it 🤣

Might need to tidy up some of those names